tests/test_testing.py

```diff
@@ -31,53 +31,53 @@ def x():
         return 1
 
     def _(a=x):
-        expect(1).equals(1)
+        assert 1 == 1
 
     return Test(fn=_, module_name=mod)
 
 
 @test("Test.name should return the name of the function it wraps")
 def _(anonymous_test=anonymous_test):
-    expect(anonymous_test.name).equals("_")
+    assert anonymous_test.name == "_"
 
 
 @test("Test.qualified_name should return `module_name.function_name`")
 def _():
-    expect(t.qualified_name).equals(f"{mod}.{f.__name__}")
+    assert t.qualified_name == f"{mod}.{f.__name__}"
 
 
 @test("Test.qualified_name should return `module_name._` when test name is _")
 def _(anonymous_test=anonymous_test):
-    expect(anonymous_test.qualified_name).equals(f"{mod}._")
+    assert anonymous_test.qualified_name == f"{mod}._"
 
 
 @test("Test.deps should return {} when test uses no fixtures")
 def _(anonymous_test=anonymous_test):
-    expect(anonymous_test.deps()).equals({})
+    assert anonymous_test.deps() == {}
 
 
 @test("Test.deps should return correct params when test uses fixtures")
 def _(dependent_test=dependent_test):
     deps = dependent_test.deps()
-    expect(deps).contains("a")
+    assert "a" in deps
 
 
 @test("Test.has_deps should return True when test uses fixtures")
 def _(dependent_test=dependent_test):
-    expect(dependent_test.has_deps).equals(True)
+    assert dependent_test.has_deps
 
 
 @test("Test.has_deps should return False when test doesn't use fixtures")
 def _(anonymous_test=anonymous_test):
-    expect(anonymous_test.has_deps).equals(False)
+    assert not anonymous_test.has_deps
 
 
 @test("Test.__call__ should delegate to the function it wraps")
 def _():
     mock = Mock()
     t = Test(fn=mock, module_name=mod)
     t(1, 2, key="val")
-    expect(mock).called_once_with(1, 2, key="val")
+    mock.assert_called_once_with(1, 2, key="val")
```

Comment:
I'm probably just unfamiliar with all the internet functions, but this change strikes me as odd because it's out of pattern. Everything else has changed `expect(something)...` to `assert expression`, but this is the only one that doesn't?

Reply:
Thanks for looking, really appreciate you taking the time!

`mock.assert_called_once_with` is just a method in `unittest.mock.Mock`, it raises an `AssertionError` if the mock wasn't called exactly once with the args.